src/domain/usecases/ExportDataSetsUseCase.ts

```diff
@@ -36,16 +36,13 @@ export class ExportDataSetsUseCase {
                     .map(ds => {
                         const headers = ds.headers;
                         if (!headers) return ds;
+
+                        // Temporary removed ou label and period
                         const newHeaders = {
-                            healthFacility: config.ouLabel
-                                ? `${headers.healthFacility} ${config.ouLabel}`
-                                : headers.healthFacility,
-                            reportingPeriod: config.periodLabel
-                                ? `${headers.reportingPeriod} ${config.periodLabel}`
-                                : headers.reportingPeriod,
+                            healthFacility: headers.healthFacility,
+                            reportingPeriod: headers.reportingPeriod,
                         };
 
-                        // (includeHeaders ? newHeaders: undefined) previously we were always including headers
                         return ds.updateHeaders(newHeaders);
                     })
```

Comment:
Is all this equivalent to this? `.map(ds => (ds.headers ? ds.updateHeaders(ds.headers) : ds))`

Reply:
Indeed now this piece .map is useless is doing nothing actually. That was because the both responsabilities was doing this map now are gone (temporary)